rosdep/base.yaml

```diff
@@ -2986,6 +3014,7 @@ liburdfdom-dev:
   arch: [urdfdom]
   debian: [liburdfdom-dev]
   fedora: [urdfdom-devel]
+  freebsd: [ros-urdfdom]
```

Comment:
I don't see ros-urdfdom in the index: https://www.freshports.org/search.php?stype=name&method=match&query=ros-urdfdom&num=10&orderby=category&orderbyupdown=asc&search=Search&format=html&branch=head

Reply:
I have removed the ros-urdfdom changes for the moment as I would need to push those ports forward. I've got the basics done. Just need to push them forward. Without ros-urdfdom, I think you get the base communication system up and running.